webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -864,6 +865,8 @@ def test_basic(self, client):
             "results": [
                 {
                     "extra": {
+                        "crashing_thread_id": 0,
+                        "crashing_thread_name": "SomeThread",
```

Comment:
Same here. We're not making changes to signature generation, so we don't need to change this file.